TASK-6/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using TASK_6.Models;
+using TASK_6.Services;
+
+namespace TASK_6;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var listOfShopsFromJson = new ListOfShopsFromJson();
+        JsonDeserializer.GetDataFromJson(ref listOfShopsFromJson);
```

Comment:
Make 'GetDataFromJson' as 'GetShops' and return ListOfShopsFromJson(Also you can rename it as ListOfShops). 
Then do just var listOfShops = JsonDeserializer.GetShops();